DualityEditorPlugins/EditorBase/PropertyEditors/Resources/FontContentPropertyEditor.cs

```diff
@@ -27,7 +27,7 @@ protected override void OnGetValue()
 			Font font = this.GetValue().FirstOrDefault() as Font;
 
 			bool lastCanRenderFont = this.canRenderFont;
-			this.canRenderFont = (font != null && font.EmbeddedTrueTypeFont != null);
+			this.canRenderFont = (font != null && font.GetGlyphBitmap(' ') != null);

```

Comment:
@ilexp  Can you explain what you mean by "Render" in this case? In the case of a bitmap font you will never have a bitmap font, but it can still be rendered (drawn to screen) no?

The problem I had with the property grid making all properties readonly was directly related to this  because without it running, the canRenderFont was set to false and therefore all properties were set to readonly.